### ISSUE TITLE
Missing option for the second question of 5th part

### ISSUE THREAD

**Issue by FakhourMed** [DevC-Casa/stateofdev.ma#67]
Hi there, i noticed that there is (maybe) a missing option in the second question of 5th part,
"Did you ever participate in an open-source project?" , the possible answers are either i have already participated (and other stuff) or i dont have a Github account. i think that you forgot the simply " I didn't participate in any open-source project"

**Comment by aboullaite** [comment]
Fair point! we'll fix that next year hopefully. keeping this open for now

**Comment by aboullaite** [comment]
This was fixed in latest version. Closing this for now. Feel free to reopen if you do not agree
